TESTING.asciidoc

```diff
@@ -305,12 +305,6 @@ e.g. -Dtests.rest.suite=index,get,create/10_with_id
 blacklisted and need to be skipped
 e.g. -Dtests.rest.blacklist=index/*/Index document,get/10_basic/*
 
-Note that the REST tests, like all the integration tests, can be run against an external
```

Comment:
Dropping this paragraph loses that `tests.cluster` applies to the REST tests too.

Reply:
The rest tests use `tests.rest.cluster`